docs/cms/input-types.adoc

```diff
@@ -91,22 +91,42 @@ image::Content-selector-default.png[An input box with a dropdown options below i
   </config>
 </input>
 ----
-<1> *allowContentType* is used to limit the content types that may be selected using the <<allowContentType Samples>>
+<1> *allowContentType* is used to limit the content types that may be selected using the <<allowContentType>>
 <2> *allowPath* is used to limit the path of the content that may be selected using the <<allowPath Samples>>. By default, Content Selector
 only displays content from the same site.
 <3> *treeMode* toggles between tree and flat modes. Default value is `false` (flat mode). +
 image:Content-selector-tree.png[Input box with a dropdown with a tree structure, 350]
 <4> *hideToggleIcon* hides the toggle icon when set to `false`. +
 image:Content-selector-no-icon.png[Input box without an icon, 500]
 
-=== allowContentType Samples
+[#allowContentType]
+=== Allow content type pattern
+There are two modes of parsing content type patterns in XP: `LEGACY` had been used before image:../images/xp-770.svg[XP 7.7.0,opts=inline] and `MATCH` is current default mode. It can be switched back in <<../deployment/config.adoc#admin, config>>.
```

Comment:
```suggestion
There are two modes of matching content type patterns in XP: `LEGACY` to simulate pattern matching before image:../images/xp-770.svg[XP 7.7.0,opts=inline] and `MATCH` as current default mode. It can be configured in <<../deployment/config.adoc#admin, config>>.
```